build-scripts/runtime_lib.cmake

```diff
@@ -78,6 +78,12 @@ elseif (WAMR_BUILD_LIBC_WASI EQUAL 1)
     include (${IWASM_DIR}/libraries/libc-wasi/libc_wasi.cmake)
 endif ()
 
+if (WAMR_BUILD_LIB_PTHREAD_SEMAPHORE EQUAL 1)
+    include (${IWASM_DIR}/libraries/lib-pthread/lib_pthread.cmake)
```

Comment:
No need to include lib_pthread.cmake here, as L88 will include it.

Reply:
And shall we add a prompt in config_common.cmake, e.g. after the following lines:
```cmake
if (WAMR_BUILD_LIB_PTHREAD EQUAL 1)
  message ("     Lib pthread enabled")
endif ()
```
Add:
```
if (WAMR_BUILD_LIB_PTHREAD_SEMAPHORE EQUAL 1)
  message ("     Lib pthread semaphore enabled")
endif ()
```